fog/view/server/src/router_request_handler.rs

```diff
@@ -233,7 +233,7 @@ async fn authenticate_view_store<E: ViewEnclaveProxy>(
     logger: Logger,
 ) -> Result<(), RouterServerError> {
     let view_store_id = view_store_url.responder_id()?;
-    let client_auth_request = enclave.view_store_init(view_store_id.clone())?;
+    let nonce_auth_request = enclave.view_store_init(view_store_id.clone())?;
```

Comment:
:question: Whats the reason for the `nonce_` prefix.  Would this be more stable to be `auth_request`.  I'm coming in fresh, so may not fully understand the Nonce types and usage, but it feels like `nonce_` is the data type and ideally this code shouldn't need a change if the underlying data types change.

Similar thought on the `nonce_auth_response` name

Reply:
I agree with Nick here.

Reply:
The `nonce` helps the reader understand that it corresponds to an actual type `NonceAuthRequest` rather than a nebulous `auth_request` that might be passed to a grpc call. This shouldn't get changed again so I don't see the staleness being an issue.

I'd prefer to keep it because I often have a hard time understanding what the types are in the handler because we're doing so many conversions between bytes and structs. That said, if you both feel strongly, I'll change it.

Reply:
I'm fine with leaving it and see how it plays out.

Reply:
Yeah my feelings are not that strong.

Reply:
FWIW, I don't think there's any value in calling this `nonce_auth_response`. There's only one type of auth that's used in this entire function. ;-)